Makefile

```diff
@@ -1052,10 +1052,12 @@ ifeq ($(ENABLE_PYOSYS),1)
 	$(INSTALL_SUDO) cp libyosys.so $(DESTDIR)$(PYTHON_DESTDIR)/$(subst -,_,$(PROGRAM_PREFIX))pyosys/libyosys.so
 	$(INSTALL_SUDO) cp -r share $(DESTDIR)$(PYTHON_DESTDIR)/$(subst -,_,$(PROGRAM_PREFIX))pyosys
 ifeq ($(ENABLE_ABC),1)
+ifeq ($(ABCEXTERNAL),)
 	$(INSTALL_SUDO) cp yosys-abc $(DESTDIR)$(PYTHON_DESTDIR)/$(subst -,_,$(PROGRAM_PREFIX))pyosys/yosys-abc
```

Comment:
Could adjust the `cp` source to also use `$(PROGRAM_PREFIX)`? (and potentially also `$(EXE)` but I think that one is less important)